src/main/java/hw_7/Figure.java

```diff
@@ -0,0 +1,9 @@
+package hw_7;
+
+public abstract class Figure {
+    public abstract double getPerimeter();
+    public abstract double getArea();
+
+    @Override
+    public abstract String toString();
```

Comment:
I guess this method from this interface  can be removed

Reply:
done